src/follower.py

```diff
@@ -98,17 +100,26 @@ def convert_camera_angle(self):
         else:
             self._turn_angle = 90
 
-
     """
     Resets the camera to the default position.
 
     The default position is tilted to 120 degrees and panned to 90 degrees.
     """
+    
```

Comment:
Can you get rid of this blank line between doc strings and function names? Here and everywhere else. I don't think it's in PEP8 and even if it is, it's not consistent with the rest of our files.

Reply:
got rid of it